<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="channelbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Channels</string>
    </property>
    <widget class="QTabWidget" name="channeltabs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="vout_tab">
      <attribute name="title">
       <string>VClamp out</string>
      </attribute>
      <widget class="QWidget" name="vout_tab_content" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>214</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vout_chan"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vout_range"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vout_reference"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gain (mV/V)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="vout_gain">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Holding position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="vout_offset">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="vout_offset_read">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Read from V in</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="cout_tab">
      <attribute name="title">
       <string>CClamp out</string>
      </attribute>
      <widget class="QWidget" name="cout_tab_content" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>214</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cout_chan"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cout_range"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cout_reference"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Gain (nA/V)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="cout_gain">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="cout_offset"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="cin_tab">
      <attribute name="title">
       <string>Current in</string>
      </attribute>
      <widget class="QWidget" name="cin_tab_content" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>214</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cin_chan"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cin_range"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cin_reference"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Gain (nA/V)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="cin_gain">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="cin_offset">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="vin_tab">
      <attribute name="title">
       <string>Voltage in</string>
      </attribute>
      <widget class="QWidget" name="vin_tab_content" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>214</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vin_chan"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vin_range"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vin_reference"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Gain (mV/V)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="vin_gain">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="vin_offset"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="channelbuttons" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>211</width>
       <height>33</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>210</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QPushButton" name="channels_reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="channels_apply">
         <property name="text">
          <string>Save changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>channelbuttons</zorder>
    <zorder>channeltabs</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulations</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>201</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QPushButton" name="simparams_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="simparams_apply">
        <property name="text">
         <string>Save changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>41</y>
       <width>287</width>
       <height>238</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelfile"/>
        </item>
        <item>
         <widget class="QToolButton" name="modelfile_browse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Population size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="npop">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Time step (ms)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dt">
          <property name="maximum">
           <double>9.990000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.250000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>VC waveforms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vc_waveforms"/>
        </item>
        <item>
         <widget class="QToolButton" name="vc_waveforms_browse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Sigma file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sigfile"/>
        </item>
        <item>
         <widget class="QToolButton" name="sigfile_browse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Data output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outdir"/>
        </item>
        <item>
         <widget class="QToolButton" name="outdir_browse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_2</zorder>
   </widget>
   <widget class="QPushButton" name="vclamp_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>194</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start voltage clamp fitting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vclamp_stop">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <zorder>channelbox</zorder>
   <zorder>groupBox</zorder>
   <zorder>vclamp_start</zorder>
   <zorder>vclamp_stop</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>channeltabs</tabstop>
  <tabstop>vout_chan</tabstop>
  <tabstop>vout_range</tabstop>
  <tabstop>vout_reference</tabstop>
  <tabstop>vout_gain</tabstop>
  <tabstop>vout_offset</tabstop>
  <tabstop>vout_offset_read</tabstop>
  <tabstop>cout_chan</tabstop>
  <tabstop>cout_range</tabstop>
  <tabstop>cout_reference</tabstop>
  <tabstop>cout_gain</tabstop>
  <tabstop>cout_offset</tabstop>
  <tabstop>cin_chan</tabstop>
  <tabstop>cin_range</tabstop>
  <tabstop>cin_reference</tabstop>
  <tabstop>cin_gain</tabstop>
  <tabstop>cin_offset</tabstop>
  <tabstop>vin_chan</tabstop>
  <tabstop>vin_range</tabstop>
  <tabstop>vin_reference</tabstop>
  <tabstop>vin_gain</tabstop>
  <tabstop>vin_offset</tabstop>
  <tabstop>channels_apply</tabstop>
  <tabstop>channels_reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
